ProfileManager.js

```diff
@@ -455,24 +456,19 @@ export default class ProfileManager {
     return userDoc.content;
   }
 
-  async updateUser({profileId, usersReferenceId, content}) {
+  async updateUser({profileAgent, usersReferenceId, content}) {
     if(!usersReferenceId) {
       usersReferenceId = edvs.getReferenceId('users');
     }
-    const {edv: usersEdv, invocationSigner} = await this.getProfileEdv({
-      profileId,
-      referenceId: usersReferenceId
-    });
-    const [userDoc] = await usersEdv.find({
+    const {edvClient, invocationSigner} = await this.getUsersEdv(
```

Comment:
Since this is a refactor PR can we go one step further and rename `invocationSigner` to `profileInvocationSigner` for added clarity.

Reply:
Strike my comment.